crypto/keyring/keyring.go

```diff
@@ -498,12 +501,16 @@ func (ks keystore) NewMnemonic(uid string, language Language, hdPath string, alg
 		return nil, "", err
 	}
 
-	info, err := ks.NewAccount(uid, mnemonic, DefaultBIP39Passphrase, hdPath, algo)
+	if passphrase == "default" {
```

Comment:
Mmmm. Why don't we rely on the zero value `""`?

Reply:
in case the default prefix passphrase changes

Reply:
so I think we'll need to modify the command line interfaces as well, correct?